app/js/components/LabResultEntry/LabResultEntry.jsx

```diff
@@ -1,17 +1,211 @@
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
 import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
+import moment from 'moment';
+import { Link } from 'react-router-dom';
+import swal from 'sweetalert';
+
+import { CustomDatePicker, PatientHeader } from '@openmrs/react-components';
+import './LabResultEntry.scss';
+import patientAction from '../../actions/patientAction';
+
+
+export class LabResultEntry extends PureComponent {
+  state = {
+    patientHeaderDetail: false,
+  }
+
+  componentDidMount() {
+    const { dispatch, history } = this.props;
+    dispatch(patientAction.getPatient(history.location.state.patient.uuid));
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const { patientHeaderDetail } = this.state;
+    if (nextProps.patientHeaderDetail.uuid !== patientHeaderDetail.uuid) {
+      this.setState({
+        patientHeaderDetail: nextProps.patientHeaderDetail,
+      });
+    }
+  }
+
+  saveEncounter = async () => {
+    const { history } = this.props;
+    const saveEncounter = await swal("Are you sure you want to save this encounter ?", {
+      buttons: {
+        YES: "YES",
+        NO: 'NO',
+      },
+    });
+    if (saveEncounter === "YES") {
+      swal("This encounter has been saved!", {
+        icon: "success",
+      }).then(() => history.push('/'));
+    }
+  }
 
-class LabResultEntry extends PureComponent {
   render() {
-    const { location } = this.props;
-    const { patient } = location.state;
-    const displayText = `Lab Results for ${patient.display}`;
+    const { patientHeaderDetail } = this.state;
+    const mockData = [{
+      name: 'Hematocrit',
+      value: 48,
+      measurement: '%',
+      range: '45% - 53%',
+    }, {
+      name: 'Platcelets',
+      value: 524000,
+      measurement: '10**3/mL',
+      range: '150,000 - 450,000',
+    }, {
+      name: 'Hematocrit',
+      value: 0,
+      measurement: '10`3/uL',
+      range: '4,500 - 11,00',
+    },
+    ];
     return (
-      <div>
-        <h1>{displayText}</h1>
+      <div className="container-fluid">
+        {patientHeaderDetail && <PatientHeader patient={patientHeaderDetail} />}
+        <h1>Test Results</h1>
+        <div className="fieldset-container">
+          <fieldset>
+            <legend>Specimen Details:</legend>
+            <div className="col-xs-12">
+              <CustomDatePicker
+                handleDateChange={() => {}}
+                labelClassName="date-picker-label"
+                label="Specimen Collection Date:"
+                defaultDate={moment().subtract(7, 'days')}
+                field="dateFromField"
+              />
+            </div>
```

Comment:
Once this ticket is implemented, we will want to replace this CustomDatePicker with the new EncounterDate component (which will likely use the CustomDatePicker behind the scenes).  

(this is just an fyi, no action required here)